axelrod/tests/integration/test_tournament.py

```diff
@@ -30,8 +31,10 @@ def setUpClass(cls):
         cls.expected_outcome.sort()
 
     def test_full_tournament(self):
```

Comment:
How much does this test affect things? If possible I'd like to keep a full tournament test **but** if there's a big speed up could we use the [random tournament property](https://github.com/Axelrod-Python/Axelrod/blob/master/axelrod/tests/property.py#L69) (which implements this as a hypothesis "strategy"), this would ensure we don't have issues with hunting down seeds and would also shrink failures etc...

The code would look something like:

```
from axelrod.tests.property import tournaments

@given(tournaments(
    strategies=short_run_time_strategies,
    min_size=1,
    max_size=30,
    min_turns=20,
    max_turns=200,
    min_noise=0,
    max_noise=1,
    min_repetitions=1,
    max_repetitions=20,
)
@settings(max_examples=5)
def test_big_tournaments(tournament):
        self.assertIsNone(
            tournament.play(progress_bar=False, filename=filename, build_results=False)
)
```

Reply:
It helps a lot, many by excluding the meta strategies and reducing the total players. I'm fine with the hypothesis way of doing it

Reply:
Cool, let's go with removing the full test then :)